packages/roosterjs-editor-plugins/lib/plugins/ContentEdit/utils/getAutoNumberingListStyle.ts

```diff
@@ -90,16 +90,19 @@ const DecimalsTypes: Record<number, number> = {
     [Character.DoubleParenthesis]: NumberingListType.DecimalDoubleParenthesis,
 };
 
-const identifyNumberingListType = (numbering: string): NumberingListType | null => {
-    // If the marker length is 3, the marker style is double parenthis such as (1), (A). Then the number is second character of the string and the separator is first character and last character.
-    const separator = numbering.length === 3 ? numbering[0] : numbering[1];
-    const secondSeparator = numbering.length === 3 ? numbering[2] : undefined;
-    const char = identifyCharacter(separator, secondSeparator);
+const identifyNumberingListType = (
+    numbering: string,
+    isDoubleParenthesis: boolean,
+    num?: number
+): NumberingListType | null => {
+    const separatorCharacter = isDoubleParenthesis
+        ? Character.DoubleParenthesis
+        : characters[numbering[1]];
     // if separator is not valid, no need to check if the number is valid.
-    if (char) {
+    if (separatorCharacter) {
         const number = numbering.length === 3 ? numbering[1] : numbering[0];
-        const numberingType = identifyNumberingType(number);
-        return numberingType ? numberingListTypes[numberingType](char) : null;
+        const numberingType = num ? identifyNumberingType(number) : numberingTriggers[number];
```

Comment:
How to do you want to handle num == 0?

Reply:
0 can treated as undefined, if num === 0 it means the list must start from beginning, so we should only check for a, A, i, I and 1, them numberingTrigger is enough

Reply:
ok